server/views/map/show.jade

```diff
@@ -1,37 +1,37 @@
 extends ../layout-wide
 block content
         .text-center.map-fixed-header
-            h1 Challenge Map
             p Required challenges are marked with a *
             .row
                 .col-xs-10.col-xs-offset-1.col-sm-6.col-sm-offset-3.col-md-4.col-md-offset-4
                     .btn.btn-primary.btn-block.active#showAll Collapse all challenges
             hr
-        #accordion.map-accordion
```

Comment:
@SaintPeter have you tested auto-scroll? Because it depends on the class `map-accordion`. https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/client/main.js#L282